pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/legacy/fraser/TBU1201_Generated.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.pensjon.brev.maler.legacy.fraser
+
+import no.nav.pensjon.brev.template.*
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.dsl.*
+
+object TBU1201_Generated : OutlinePhrase<LangBokmalNynorskEnglish>() {
```

Comment:
Formattering i mange av filene her ser ut til å ha litt for mye indentering.

Reply:
Har du noen eksempler? Denne fila så grei ut for meg. Jeg har generert koden med tabs og ikke spaces, så om du har noe annet enn 4 spaces pr tab så vil det se rart ut.